lib/merch_calendar/retail_calendar.rb

```diff
@@ -27,7 +27,7 @@ def start_of_year(year)
     # The starting date of a given month
     # THIS IS THE MERCH MONTH
     # 1 = feb
-    # 
+
```

Comment:
Comments about a method should be right on top of the method, no blank lines in between.